MIGRATION_GUIDE.md

```diff
@@ -115,6 +115,20 @@ APIs. A similar result can be achieved by using `waitFor`:
 +});
 ```
 
+Alternatively this utility can be implemented as the following, for a direct migration:
+
+```ts
+import { waitFor } from '@testing-library/react';
+
+const waitForValueToChange = async <T,>(getValue: () => T) => {
```

Comment:
**nitpick:** I don't think the "," in `<T,>` is doing anything, right?

Reply:
Apologies, you're right! I had this written in a `.tsx` file where the `,` was necessary to differentiate from a jsx tag.

```suggestion
const waitForValueToChange = async <T>(getValue: () => T) => {
```